charts/jenkins/values.yaml

```
@@ -404,6 +404,28 @@ master:
     #   hosts:
     #     - jenkins.cluster.local
 
+  # often you want to have your controller all locked down and private
+  # but you still want to get webhooks from your SCM
+  # A secondary ingress will let you expose different urls
+  # with a differnt configuration
+  secondaryingress:
+    enabled: false
+    # paths you want forwarded to the backend
+    # ex /github-webhook
+    paths: []
```

what about putting the github webhook in here by default commented out?

I suspect that would be what it's used for the majority of the time

```suggestion
    paths: []
    #paths:
    # - /github-webhook
```

---

does helm merge or overwrite arrays?
if its merge, that would mean there's no way to disable it

What about the straight up non github webhook? I think its /notifyCommit
What about prefixes? by making it need to be configured, its able to be prefixed, the default wouldn't be.

---

ah, it overwrites a list, so it would be doable.

---

I had it commented out to avoid those issues 👅

---

oooh i get what your saying, i can change it if desired when i'm on a computer again

---

is fine 🚢